A72-open-telemetry-tracing.md

```diff
@@ -364,22 +437,21 @@ GlobalInterceptors.setInterceptors(
 ```
 
 #### C++
-In C++, we will add a new method in `OpenTelemetryPluginBuilder`, see [gRPC A66][A66].
+The following new methods will be added in `OpenTelemetryPluginBuilder`.
 
 ```C++
 class OpenTelemetryPluginBuilder {
  public:
-  OpenTelemetryPluginBuilder();
-  // If `SetMeterProvider()` is not called, no metrics are collected.
-  OpenTelemetryPluginBuilder& SetMeterProvider(
-      std::shared_ptr<opentelemetry::metrics::MeterProvider> meter_provider);
-+ // Set one or multiple propagators for span context propagation, e.g. 
-+ // GrpcTraceBinPropagator or community standard ones like W3C, etc.
-+ OpenTelemetryPluginBuilder& SetPropagator(
-+     std::shared_ptr<opentelemetry::context::propagation::TextMapPropagator> 
-+         new grpc::GrpcTraceBinPropagator());
-...
-}
+  // If `SetTracerProvider()` is not called, no traces are collected.
+  OpenTelemetryPluginBuilder& SetTracerProvider(
+      std::shared_ptr<opentelemetry::metrics::SetTracerProvider>
+          tracer_provider);
+  // Set one or multiple text map propagators for span context propagation, e.g.
+  // GrpcTraceBinPropagator or community standard ones like W3C, etc.
```

Comment:
not your fault but the `GrpcTraceBinPropagator ` should be `GrpcTraceBinTextMapPropagator` to match elsewhere

Reply:
done